bindings/wasm/fixup.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+import sys
+
+if len(sys.argv) != 2:
+    print("Usage: python fixup.py <SOURCE_DIR>")
+
+filename = sys.argv[1]
+with open(filename, "r") as file:
+    data = file.read()
+
+data = data.replace(
+    'var workerOptions={type:"module",workerData:"em-pthread",name:"em-pthread"};', ""
+)
+data = data.replace(
+    "workerOptions", '{type:"module",workerData:"em-pthread",name:"em-pthread"}'
```

Comment:
What is this?

Reply:
emscripten generating some non-static worker options that Vite cannot handle, and it can be trivially modified to become static.